src/utils/api.js

```diff
@@ -1,4 +1,18 @@
-// handles api requests with configurable parameters and callback functions
+
+async function loadConfig() {
```

Comment:
This is something that I would call once on Vue app load, and place the values in the store. Then different components can just get what they need from the store, i.e. getBackendUrl or whatever. That way you aren't making the fetch of the file more than one time if its needed in multiple places.

Reply:
I see! Okey dokes, gonna do that!